nixos/modules/services/desktops/geoclue2.nix

```
@@ -179,6 +181,12 @@ in
 
   ###### implementation
   config = mkIf cfg.enable {
+    warnings = optional (cfg.geoProviderUrl == mozillaLocationServices)
```

```suggestion
    warnings = optional ((cfg.enableWifi || cfg.enable3G) && cfg.geoProviderUrl == mozillaLocationServices)
```

That URL is only used for the `gclue-3g` and `gclue-wifi` sources, so the warning should be gated accordingly.